spec/models/team_member_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe TeamMember, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I think you can remove this empty spec file according to the automatically generated notice (“add some examples to (**or delete**) …”).

Reply:
I was thinking that this file you be useful later on, but for now it is definitely cleaner to delete it.